### ISSUE TITLE
Telegram always performs sync after update

### ISSUE THREAD

**Issue by UKPhil** [ubports/telegram-app#153]
Following the latest update (2.5.3.2) I have noy been able to use Telegram. 

I teceive the notifivations all OK and they are cleared down correct when I view messages on another device. Howe vet, when I go into the message on my OnePlus One device the sysyem is always performing a sync.

I have internet connection  all  OK.

I have tried to uninstall the appearance and then install again but this does not work. When I load the appearance after a new install, I am taken to the existing version. I am unable to lohout anf put new details in.

This is the error log;
http://paste.ubuntu.com/p/PGv3VZd8GZ/

**Comment by UKPhil** [comment]
Just noticed also when I swipe from the sside, there is a space for the Telegram preview window but it is just a space and no Telegram preview can be seen.

**Comment by UKPhil** [comment]
As advised by @hummlbach, I have since deleted the two directories .cache/com.ubuntu.telegram and .config/com.ubuntu.telegram in my home directory and installed a fresh version. THis has worked and now my Ubuntu Telegram app is back to normal.

I wonder why this originally happened though and why the app remove did not delete these folders.

One thing that I have noticed is that the web version of Telegram has far more features than the Ubuntu one and that it looks far smoother. Is there any plans to make ours look and feel like the web one?

**Comment by UKPhil** [comment]
Update, 

I was using the Telegram app all OK for a few hours and now it has happened again. I look in the error log and it is the same as before; socketerror "2" Host not found and Hostnotfound error in tcp socket.

**Comment by Flohack74** [comment]
@UKPhil so first a general rule, uninstalling an App will not remove its data folders, this is common for all UT apps. For the connection issue, can you please attach a log file for this?

BR Florian

**Comment by UKPhil** [comment]
@Flohack74 the log that you require, is this different to the link one I added in the first post?

**Comment by UKPhil** [comment]
@Flohack74  thanks for getting back to me and hopefully you will have a fix soon. I tried this on my M10 last night and all was OK but a little slow. So for me it seems to be only the OnePlus One that is experiencing this issue. 

Thanks for your help and good luck, I hope you get it fixed soon :-)

**Comment by UKPhil** [comment]
@Flohack74 I was just thinking, if it to do with regions of the world how is it that my M10 is still working all OK after the last update?

**Comment by Flohack74** [comment]
@UKPhil Are you with both devices in the same network, or one is WiFi and the other radio?

**Comment by UKPhil** [comment]
My M10 tablet at home is going through my home wifi which is with TPG  and the OnePlus One is on the Telstra network.

When I get home I will connect the OnePlus One to the home wifi and see if there is any difference.

**Comment by Flohack74** [comment]
Yes please, since that means you got 2 different IP addies, which could put you in 2 different "geo locations" since geolocation by IP is the worst thing on this planet.

**Comment by SwizzleMaster** [comment]
I'm also affected by this bug.

Phone: Nexus 5, Location: Taiwan

Steps: 
1) Clear the cache and everything else 
2) reinstall telegram.
3) not sure what this does but i cleared all the old logins from other devices/installs using the telegram desktop client.
4) sign in 

Behavior: 
1) before reinstallation, it still receives notifications even if i'm not signed in. I need to sign in to read them.
2) if a message is received while i have the chat open things work as expected. but if i tap on a notification to open, the bug happens.

Where should i pull the logs from?

**Comment by Flohack74** [comment]
No logs needed, I think I know where it comes from.

**Comment by UKPhil** [comment]
Great news so does that mean that we will have a fix soon?

**Comment by Flohack74** [comment]
Not necessarily does it mean I know how to fix it :) But I'll do my best.

**Comment by UKPhil** [comment]
@Flohack74 we know you will do your best and I thank you for that. I really look forward to deploying the fix for it soon wink wink.
